Dockerfile

```diff
@@ -77,6 +77,9 @@ ADD dev-requirements.txt /dev-requirements.txt
 ADD requirements.txt /requirements.txt
 ADD rbuild.ini /rbuild.ini
 
+# Temporarily install a new cmake until switching to ubuntu 22.04
+RUN pip3 install cmake==3.22.1
```

Comment:
ORT needs 3.26.4 would it make sense to bump this to 3.26.4 too?

Reply:
no, we should make sure our stuff builds with the cmake in ubuntu 22.04.